src/js/services/storageService.js

```diff
@@ -78,6 +78,14 @@ angular.module('copayApp.services')
     //
     // UPGRADING STORAGE
     //
+    // TODO: prior to upgrading storage a backup of all storage should be captured and made available
```

Comment:
That TODO is not a requirement; just something I thought about for the future.  I can remove the comment if you like.